client/src/modules/fiscal/fiscal.manage.js

```diff
@@ -6,24 +6,22 @@ FiscalManagementController.$inject = [
 ];
 
 /**
- * This controller is responsible for creating and updating a fiscal year. It provides
- * utility functions for submission and error handling.
+ * This controller is responsible for creating and updating a fiscal year. It
+ * provides utility functions for submission and error handling.
  */
 function FiscalManagementController($state, Fiscal, Notify, Modal, util, moment) {
-  var vm = this;
-  var id;
-  var isUpdate;
+  const vm = this;
 
-  // state variables
+  // state constiables
```

Comment:
😆 seriously `constiables` !

Reply:
I was wondering where those were coming from.  Guess it was me! :P